### DISCUSSION TITLE
Disable screen refresh for RGB panels?

### DISCUSSION THREAD

**Discussion by the-automat** [moononournation/Arduino_GFX#364] [Q&A]
I'm working with various Sunton boards (primarily 4827S043), and in general things are working great (aside: many thanks for all your hard work!). However, I'm having issues with trying to do OTA updates--the low level screen refresh seems to be interfering with the data  receive loop on the ESP32 and causing timeouts (and just plain causing long upload times even if it doesn't timeout).
Is there an existing API call to disable the screen refresh routine? The ESP32 will reboot at the conclusion of the upload, so it doesn't necessarily need to be reversible.

**Comment by moononournation** [comment]
In ESP32 general, hold the GPIO 0 down while reset can enter the download mode.

**Comment by Tenebrous** [comment]
I hit this while adding NVS persistence to an ESP32-S3 RGB/PSRAM application.

`Preferences` reads and writes panic with `Cache disabled but cached memory region accessed` while `Arduino_ESP32RGBPanel` is continuously refreshing its PSRAM framebuffer. The decoded trace reaches `esp_partition_write_raw()` through `nvs_set_u64()`.

The ESP-IDF workaround is to destroy the RGB panel before the flash operation (`esp_lcd_panel_del(panel_handle)`), then recreate it and redraw afterward. `Arduino_ESP32RGBPanel` owns that handle privately and exposes no lifecycle API, so applications cannot do this without patching the library.

Could `Arduino_ESP32RGBPanel` expose a public `end()` / `deinit()` method that calls `esp_lcd_panel_del()` and clears the handle? A paired `Arduino_RGB_Display::resume()` helper would make the recovery path safe and ergonomic. This would also help OTA, SPIFFS/LittleFS, and any other flash-backed persistence.
